pipelines/LSSTComCam/cpFlat.yaml

```diff
@@ -7,3 +7,7 @@ tasks:
     class: lsst.ip.isr.IsrTaskLSST
     config:
       doDeferredCharge: true
+  cpFlatCombine:
+    class: lsst.cp.pipe.CalibCombineByFilterTask
+    config:
+      flatSource: "TWILIGHT"
```

Comment:
Is this the only possible option for comCam? (Sorry I missed all the comCam data taking!) If not I wouldn't set it here.

Reply:
Yup.  We didn't have a flat field screen, and so we made twilight flats.  I also don't anticipate ComCam going back on the telescope anytime soon.
In a sense this is academic since we also aren't regenerating twilight flats for ComCam but I thought it would be useful to have this for completeness.